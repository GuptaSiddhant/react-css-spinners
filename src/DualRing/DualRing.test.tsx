import React from 'react'
import renderer from 'react-test-renderer'
import { DualRing } from './'

it('renders correctly', (): void => {
  const tree = renderer.create(<DualRing />).toJSON()
  expect(tree).toMatchSnapshot()
})
