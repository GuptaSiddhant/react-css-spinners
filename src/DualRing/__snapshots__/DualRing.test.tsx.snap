// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rcs-dual-ring"
  style={
    Object {
      "--rcs-dual-ring-color": "#fff",
      "--rcs-dual-ring-size": "64px",
      "--rcs-dual-ring-thickness": "5px",
    }
  }
/>
`;
